it/uk/gov/hmrc/agentassurance/controllers/PropertiesControllerISpec.scala

```diff
@@ -96,94 +79,42 @@ class PropertiesControllerISpec extends UnitSpec
     behave like updatePropertyTests("manually-assured", false)
   }
 
-  def createPropertyTests(key: String) = {
```

Comment:
Why is these tests being removed?

Reply:
we now have 1 function that does create and update, so I rewired the routing in case create is still used, but we do not have the need for it and I have removed the code for it